lib/models/release.ts

```diff
@@ -224,7 +224,6 @@ const getReleaseModel = function (
 			.sort((a, b) => a.service_name.localeCompare(b.service_name));
 
 		release.user = (release.is_created_by__user as BalenaSdk.User[])[0];
-		delete release.is_created_by__user;
```

Comment:
This actually made the typings wrong (since it should at least hold a pine deferred), especially in case the user passes some options with an `$expand` for the `is_created_by__user`